src/util/checkWalletForNetwork.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { InjectedWindow } from '@polkadot/extension-inject/types';
+import { Wallet } from '~src/types';
+
+export const checkWalletForNetwork = (network: string) => {
+	if(!network) return;
+	const isNovaWallet = (window as any).walletExtension?.isNovaWallet;
+	const isPolymeshWallet = ['polymesh'].includes(network);
```

Comment:
const isPolymeshWallet  -> isPolymeshNetwork
Won't need this variable i believe